src/storage.py

```diff
@@ -58,7 +64,11 @@ def get_arousal_labels(self):
             pandas.DataFrame: A 1D pandas dataframe with labels for arousal.
         """
         if (self._labels_arousal is not None):
-            return self._labels_arousal
+            if(self._throw_data):
```

Comment:
Add same support for get_labels() aswell.

return np.column_stack(
            (self.get_arousal_labels(), self.get_valence_labels()))

Reply:
Done!